src/v/model/fundamental.h

```diff
@@ -238,6 +238,26 @@ class materialized_ntp {
     const model::ntp& _source_or_input;
 };
 
+/**
+ * Enum describing Kafka's control record types. The Control record key schema
+ * is defined as a tuple of [version: int16_t, type: control_record_type].
+ * Currently Kafka protocol uses version 0 of control record schema.
```

Comment:
Sorry I still don't understand what `Currently Kafka protocol uses version 0 of control record schema.` means. This sentence seems to be saying that Kafka doesn't--for example--ever use `tx_commit`, but i see many uses of it and librdkafka has a case statement for that control record type.

Reply:
This sentence is only mentioning `version` of a schema, the only one supported in Kafka code is a schema with version 0

Reply:
that makes sense i completely missed that there was a version field.